lib/suspenders/cleanup/generate_readme.rb

```diff
@@ -0,0 +1,165 @@
+require_relative "../../generators/suspenders/lint_generator"
+require_relative "../../generators/suspenders/styles_generator"
+require_relative "../../generators/suspenders/testing_generator"
+require_relative "../../generators/suspenders/views_generator"
+
+module Suspenders
+  module Cleanup
+    class GenerateReadme
+      include Suspenders::Generators::Helpers
+
+      def self.perform(readme, app_name)
+        new(readme, app_name).perform
+      end
+
+      attr_reader :readme, :app_name
+
+      def initialize(readme, app_name)
+        @readme = readme
+        @app_name = app_name
+      end
+
+      def perform
+        File.open(readme, "w+") do |file|
+          @file = file
+
+          heading app_name.titleize, level: 1
+
+          prerequisites
+
+          local_development
+
+          heading "Configuration", level: 2
+
+          heading "Test", level: 3
+          description_for Suspenders::Generators::Environments::TestGenerator
+
+          heading "Development", level: 3
+          description_for Suspenders::Generators::Environments::DevelopmentGenerator
+
+          heading "Production", level: 3
+          description_for Suspenders::Generators::Environments::ProductionGenerator
+
+          heading "Linting", level: 3
+          description_for Suspenders::Generators::LintGenerator
+
+          heading "Testing", level: 2
+          description_for Suspenders::Generators::TestingGenerator
+
+          heading "Factories", level: 3
+          description_for Suspenders::Generators::FactoriesGenerator
+
+          heading "Accessibility", level: 2
+          description_for Suspenders::Generators::AccessibilityGenerator
+          content <<~MARKDOWN
+            For more information, review the [Accessibility Tooling][] section in 
+            the [CONTRIBUTING][] guide.
+
+            [Accessibility Tooling]: ./CONTRIBUTING.md#accessibility-tooling
+            [CONTRIBUTING]: ./CONTRIBUTING.md
+          MARKDOWN
+          new_line
+
+          heading "Advisories", level: 2
+          description_for Suspenders::Generators::AdvisoriesGenerator
+
+          heading "Mailers", level: 2
+          description_for Suspenders::Generators::EmailGenerator
+
+          heading "Jobs", level: 2
+          description_for Suspenders::Generators::JobsGenerator
+
+          heading "Layout and Assets", level: 2
+
+          heading "Stylesheets", level: 3
+          description_for Suspenders::Generators::StylesGenerator
+
+          heading "Inline SVG", level: 3
+          description_for Suspenders::Generators::InlineSvgGenerator
+
+          heading "Layout", level: 3
+          description_for Suspenders::Generators::ViewsGenerator
+        end
+      end
+
+      private
+
+      def content(text)
+        @file.write text
+      end
+
+      def new_line
+        @file.write "\n"
+      end
+
+      def heading(text, level:)
+        @file.write "#{"#" * level} #{text}\n"
```

Comment:
```suggestion
        @file.write "#{'#' * level} #{text}\n"
```

That takes it from impossible to read to simply hard to read.

```suggestion
        @file.write %(#{'#' * level} #{text}\n)
```

is that even easier to read?